### ISSUE TITLE
Some challenge

### ISSUE THREAD

**Issue by mdche001** [mdche001/EC601_miniproject2_mdche#1]
1. Cannot upload file >100MB with git lfs
2. After I added two dropout layers to my model, the training process became even faster than before. And I obtained 100% accuracy after about 200 images during the training process, I think it is obviously overfitting. However, I have no idea why it occured.
3. When I try to use PIL to draw some graph for my result. An error showed the PIL can not co-exists with Tensorflow. And my Tensorflow is the version based on GPU.
